### PR TITLE
(feat) KHP3-6968 : Update Tabs for Eligible Benefits and PreAuth Requests UI Layout

### PR DESCRIPTION
**Pull request by its-kios09**
## Requirements

- [ ] This PR has a title that briefly describes the work done, including the ticket number if there is a ticket.
- [ ] My work conforms to the [**OpenMRS 3.0 Styleguide**](https://om.rs/styleguide).
- [ ] I checked for feature overlap with [**existing widgets**](https://om.rs/directory).

## Screenshots
benefits

## Related Issue

see:- https://thepalladiumgroup.atlassian.net/browse/KHP3-6968

## Other

*None.*

### CODE REVIEW

File: packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx [hunk 1]
```diff
@@ -0,0 +1,24 @@
...
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.headerContainer}>
+      <h4>{t('benefitsHeader', 'Benefits')}</h4>
+      <div>
+        <h6>{t('SHIF', 'Social Health Insurance Fund')}</h6>
+        <h6>
+          <span> {t('shifNo', 'SHIF NUMBER:')}</span>
+          <span>32JFFN23B</span>
```
[the remark this answers is not in the document]
Comment (ojwanganto) [reply packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx:17]: +1

[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx:17]: @ojwanganto @donaldkibet Thanks for the guidance let me update the PR

File: packages/esm-billing-app/src/benefits-package/benefits-package.component.tsx [hunk 2]
```diff
@@ -1,10 +1,34 @@
...
+import { CardHeader } from '@openmrs/esm-patient-common-lib';
+import { useLayoutType } from '@openmrs/esm-framework';
+import Benefits from './benefits/benefits.component';
 
 const BenefitsPackage = () => {
+  const [activeIndex, setActiveIndex] = useState(0);
+  const isTablet = useLayoutType() === 'tablet';
+  const { t } = useTranslation();
+
+  const switchTabs = [
```
[the remark this answers is not in the document]
Comment (its-kios09) [reply packages/esm-billing-app/src/benefits-package/benefits-package.component.tsx:15]: Sorted changed to Tabs

File: packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+import { useTranslation } from 'react-i18next';
+
+interface BenefitsHeaderProps {}
+
+const BenefitsHeader: React.FC<BenefitsHeaderProps> = () => {
+  const { t } = useTranslation();
+
+  return (
+    <div className={styles.headerContainer}>
+      <h4>{t('benefitsHeader', 'Benefits')}</h4>
```
Comment (makombe) [inline packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx:12]:
```suggestion
      <h4>{t('benefits', 'Benefits')}</h4>
```

File: packages/esm-billing-app/src/benefits-package/benefits/benefits.component.tsx [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+      </Accordion>
+      <Accordion>
+        <AccordionItem title="Family Planning">
+          <p>
+            Lorem ipsum dolor sit amet consectetur, adipisicing elit. Molestiae deleniti nesciunt placeat animi
+            voluptates dolor explicabo provident, quasi voluptatum! Architecto ut aut temporibus illum eos! Sit soluta
+            delectus laborum ea.
+          </p>
+        </AccordionItem>
+      </Accordion>
```
Comment (makombe) [inline packages/esm-billing-app/src/benefits-package/benefits/benefits.component.tsx:56]: Remember to remove the mock here when actual data is available

**Comment by its-kios09** [comment]
@ojwanganto @donaldkibet @makombe 

This is the changes made after the PR updates 
benefitsPreAuth

**Comment by ojwanganto** [comment]
Let's add a search feature to ease finding of benefits.

**Comment by Omoshlawi** [comment]
For the nav link "Benefit preauths" could you please rename it to something more deserving?

**Comment by its-kios09** [comment]
@ojwanganto I have add the search functionality 

benefits portal

**Comment by its-kios09** [comment]
@Omoshlawi I have changed it to Benefits portal

**Comment by ojwanganto** [comment]
> @Omoshlawi I have changed it to Benefits portal

Could we try 'Insurance benefits'?

**Comment by its-kios09** [comment]
> Insurance benefits'
@ojwanganto 
insurancebenefits

File: packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+  const shifIdentifiers = useShifIdentifiersData();
+
+  return (
+    <div className={styles.headerContainer}>
+      <h4>{t('benefits', 'Benefits')}</h4>
+      <div>
+        <h6>{t('SHIF', 'Social Health Insurance Fund')}</h6>
+        {shifIdentifiers.map((identifier) => (
+          <h6 key={identifier.identiferNumber}>
+            <span>{t('shifNo', 'SHIF NUMBER: ')}</span>
```
Comment (ojwanganto) [inline packages/esm-billing-app/src/benefits-package/benefits/benefits-header.component.tsx:19]: This should be SHA number

**Comment by ojwanganto** [comment]
Thanks for the work. Over to @donaldkibet @makombe for syntax checks

**Review by donaldkibet — APPROVED** [review]
Thanks @its-kios09

**Pull request merged**